processor/src/main/resources/application.yaml

```diff
@@ -15,6 +15,7 @@ kafka:
   consumer:
     group-id: songsapp
   topic-upload: resource-upload
```

Comment:
I usually do `kafka.topics.upload = resource-upload`

Reply:
replaced with topics.upload and topics.processed